src/invidious/channels/about.cr

```diff
@@ -99,9 +84,9 @@ def get_about_info(ucid, locale)
     description = initdata["metadata"]["channelMetadataRenderer"]?.try &.["description"]?.try &.as_s? || ""
     description_html = HTML.escape(description).gsub("\n", "<br>")
 
-    paid = about.xpath_node(%q(//meta[@itemprop="paid"])).not_nil!["content"] == "True"
-    is_family_friendly = about.xpath_node(%q(//meta[@itemprop="isFamilyFriendly"])).not_nil!["content"] == "True"
-    allowed_regions = about.xpath_node(%q(//meta[@itemprop="regionsAllowed"])).not_nil!["content"].split(",")
+    paid = false
```

Comment:
How are we going to know if a channel is paid or not? Did youtube remove the info?

Reply:
Well, I tried my best to find a paid channel, and I found nothing and on the innertube API there is no info about it. I think paid channels got removed a long time ago.
This info still exist on the HTML `about` page of a channel though for backward compatibilities.

Reply:
In that case why not just remove the paid attribute? YouTube has deprecated it and Invidious doesn't even use it anywhere besides in the serialization methods.